### ISSUE TITLE
Empty table header in Search Results table

### ISSUE THREAD

**Issue by echappen** [ustaxcourt/ef-cms#594]
The first table header (`<th>`) in Search Results table contains no text.

#### Why It Matters
The `<th>` element helps associate table cells with the correct row/column headers. A `<th>` that contains no text may result in cells with missing or incorrect header information.

#### How to Fix It
If the table cell is a header, provide text within the cell that describes the column or row. If the cell is not a header or must remain empty (such as the top-left cell in a data table), make the cell a `<td>` rather than a `<th>`.

#### Steps to Reproduce 
- Navigate to homepage
- Click Order tab
- search orders by keyword "order"
- View search results

**Comment by JessicaMarine** [comment]
Moved to Flexion: https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/8105
